ShenkinStore/Controllers/TransactionsController.cs

```diff
@@ -2,29 +2,73 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.Rendering;
 using Microsoft.EntityFrameworkCore;
 using ShenkinStore.Models;
+using ShenkinStore.ViewModels;
 
 namespace ShenkinStore.Controllers
 {
     public class TransactionsController : Controller
     {
-        private readonly ShenkinContext _context;
+        private ShenkinContext db = new ShenkinContext();
 
         public TransactionsController(ShenkinContext context)
         {
-            _context = context;
+            db = context;
         }
 
         // GET: Transactions
         public async Task<IActionResult> Index()
         {
-            var Transaction = _context.Transaction.Include(t => t.Product).Include(t => t.User);
-            return View(await Transaction.ToListAsync());
+            return View(await db.Transactions.ToListAsync());
         }
 
+        // GET: Transactions/AddressAndPayment/
+        public ActionResult AddressAndPayment()
```

Comment:
Didn't understand what's happening here... can you separate it to smaller more readable functions?